app/Model/CoPetition.php

```diff
@@ -158,7 +158,11 @@ class CoPetition extends AppModel {
                                       PetitionStatusEnum::PendingApproval,
                                       PetitionStatusEnum::PendingConfirmation)),
       'required' => true,
-      'message' => 'A valid status must be selected'
+      'message' => 'A valid status must be selected',
+      'coef_next_step' => array(
```

Comment:
`coef_next_step` need to be a distinct validation rule and not part of the `status` field rule